pages/linux/netselect.md

```diff
@@ -0,0 +1,24 @@
+# netselect
+
+> Speed test for choosing a fast network server.
+> More information: <https://github.com/apenwarr/netselect> .
+
+- List available options:
```

Comment:
Perhaps this would be best as the last option? Generally we put examples in the order you might use  them - so I'd assume that the user would want to use example #2 first instead of this one.

Reply:
Right, I'll change the order.